src/sidebar/components/HelpPanel.tsx

```diff
@@ -1,27 +1,29 @@
 import { Icon, Link, LinkButton } from '@hypothesis/frontend-shared';
+import type { ComponentChildren as Children } from 'preact';
 import { useCallback, useMemo, useState } from 'preact/hooks';
 
+import type { AuthState } from '../helpers/version-data';
 import { useSidebarStore } from '../store';
+import type { SessionService } from '../services/session';
```

Comment:
Using `import type` allows tools like Rollup to know that this is a purely compile-time dependency, without having to deeply analyze the target module.

Reply:
> You think we should interleave type imports with module imports?

This seemed to me like the simplest thing to do. I'm not aware of any strong conventions around this. We can discuss.